backend/src/main/java/com/github/EazyErik/Controller/DrinksController.java

```diff
@@ -0,0 +1,17 @@
+package com.github.EazyErik.Controller;
+
+
+import com.github.EazyErik.Service.DrinksService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController("DrinksController")
+@RequestMapping("api/addToFav")
+@RequiredArgsConstructor
+public class DrinksController {
+
+
+    public DrinksService drinksService;
```

Comment:
Fields should be private. Also this should be final. otherwise `@RequiredArgsConstructor` won't work.